rules/typescript.js

```diff
@@ -250,9 +250,9 @@ module.exports = {
     '@typescript-eslint/prefer-nullish-coalescing': [
       'error',
       {
-        forceSuggestionFixer: false,
```

Comment:
This is the cause of the issue, as this option was [removed](https://github.com/typescript-eslint/typescript-eslint/pull/5835)